### ISSUE TITLE
repeat of #139 (bytecode error)

### ISSUE THREAD

**Issue by olagarde** [TeamWizardry/Wizardry#160]
OSX, CENT7
JRE 1.7, 1.8, 1.10
Wizardry 0.9.1 through 0.9.3

Corrupted install is highly unlikely, particularly since there are multiple mods in multiple packs using the same bytecode pregeneration techniques during mod init.  It might still be something in my Java environment but I just can't find it.

- all downloads (mods, prereq's, java, minecraft core, forge, etc., etc) have correct checksums
- tried prior 3 releases in JRE 1.8 stack, same crash
- tried updating to latest 1.8 JRE, same crash
- tried 1.10 JRE compat STI for 1.8, same crash
- same with exactly one instance of JRE
- same with exactly one instance of JDK
- same with Oracle and Open JDK
- same with Oracle and Open Java runtime only
- same in fresh install on OSX and CENT7
- same in installs on base system and in fresh vbox images

For all failure cases the backtrace ends with javac bytecode validator claiming an invalid goto ref for poweredthingies pre-init bytecode generation, but this shouldn't happen if even one other user is successfully loading (which they are). I'm at a loss.

**Comment by olagarde** [comment]
I can try downgrading that far, but it would raise a couple of concerns...
1.  JDK-8046233 was reported fixed three years ago (ditto downstream push to JDK/JRE/STI 1.10).
2.  I hit the same issue with the base JRE 1.8, which precedes case 8046233.
3.  I'd be choosing between using the mod and using a javac with double-digit CVEs against it.

Version verification:  done via the "blow all java away and install next version only" approach.  Also the debug log contains the exec ref, which I can verify is a symlink pointing to the currently installed version.

**Comment by pau101** [comment]
There may very well be a regression, but I would first like to address if 8u25 resolves this problem as I would expect.

**Comment by pau101** [comment]
I would also accept any crash report you've experienced of this issue using a JRE later than what I have previously listed.

**Comment by olagarde** [comment]
Fixed, sort of. The fact that jdk versions outside 1.8 worked at all implies (requires, really) that there was a 1.8 dynlib orphaned ... somewhere. Otherwise the static forge prereq would fail, right? So I manually restored the /Library/.../JavaVM.frameworks/ and /System/.../Java/ from prior to the issue, rebooted, upgraded to 1.8.181, rebooted, and it started working again. So it probably was mixed versions, and definitely involved the base java loader dynlib (probably in /System/.../Java/ somewhere). Beyond that, who knows?
